### PR TITLE
Periodically fetch kube status

### LINKED ISSUE #964: [Feature Request]: Refresh User Permissions Periodically
### Feature description

Today we do not fetch the status endpoint more than once. This endpoint tells us your permission level and if you're part of the allowed group.

We should fetch to get the user's permissions to see if they got more or less permissions since the initial load time. 

Poll the status endpoint at the standard interval.

### Describe alternatives you've considered

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
## Description

Close #964 
Fetch the status endpoint at the regular `POLL_INTERVAL`.

## How Has This Been Tested?

Login to the dashboard -> Go to the developer console and open the Network tab -> Check if the `status` endpoint is called every 30 seconds.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by DaoDaoNoCode** [comment]
There is still a small issue here related to this PR. I found all the hooks in `App.tsx` will be called twice, so the `status` endpoint will also be called twice simultaneously every 30 seconds.

File: frontend/src/redux/actions/actions.ts [hunk 1]
```diff
@@ -41,16 +42,18 @@ export const getUserRejected = (error: Error): GetUserAction => ({
 
 export const detectUser = (): ThunkAction<void, AppState, unknown, Action<string>> => {
```
Comment (andrewballantyne) [inline frontend/src/redux/actions/actions.ts:43]: I get detectUser is a thunk -- but perhaps we need to turn it into an effect, that uses a dispatch to trigger it after it comes back. That way we can clean up the effect when it gets remounted / recalled

**Comment by Gkrumbach07** [comment]
Double render is gone, and hook cleans up properly

//lhtm

**Review by andrewballantyne — APPROVED** [review]
Tested it on cluster -- snappy feedback; max 30s until revoked.

**Pull request merged**